assembl/alembic/env.py

```diff
@@ -23,8 +24,9 @@
 set_config(config.file_config._sections['app:assembl'])
 pyramid_env = bootstrap(config.config_file_name)
 configure_zmq(pyramid_env['registry'].settings['changes.socket'], False)
-
 configure_engine(pyramid_env['registry'].settings, False)
+configure_indexing(pyramid_env['registry'].settings)
```

Comment:
Adding this line here means that elasticsearch will be configured at initialization of each migration, right? If yes, why does it need to?